server/graphql/general/src/queries/plugin.rs

```diff
@@ -0,0 +1,25 @@
+use async_graphql::*;
+use graphql_core::ContextExt;
```

Comment:
```suggestion
use graphql_core::{standard_graphql_error::StandardGraphqlError, ContextExt};
```

(Belongs to the comment below)